sql/7_0_1-to-7_0_2_upgrade.sql

```diff
@@ -577,4 +577,11 @@ ALTER TABLE `x12_partners` ADD COLUMN `x12_claim_status_endpoint` tinytext;
 
 #IfMissingColumn x12_partners x12_attachment_endpoint
 ALTER TABLE `x12_partners` ADD COLUMN `x12_attachment_endpoint` tinytext;
+#EndIf
+
+#IfRow lists type allergy
+SET @username = (SELECT `username` FROM `users` WHERE `authorized` = 1 AND `active` = 1 LIMIT 1);
+UPDATE `lists` SET `user` = @username WHERE `type` = 'allergy' AND (`user` = '' OR `user` IS NULL);
+SET @user = (SELECT `user` FROM `lists` WHERE `type` = 'allergy');
```

Comment:
looks like will have to create a special function in `SQLUpgradeService` to update each user who's `npi` is empty since this will fail when the subquery returns more than 1 row